dokka-integration-tests/gradle/projects/it-multiplatform-multimodule/second/src/commonMain/kotlin/Usage.kt

```diff
@@ -0,0 +1,10 @@
+package it.mpp.second
+
+import it.mpp.first.*
+
+/**
+ * exposes [Subclass] from another module which has [SubclassOptInRequired] annotation present
+ */
+class Usage(
```

Comment:
Test failes when generating documentation for this module `second` which depends on `first`.

It's not possible to write just a unit test for this, as we need to first compile `first` and then generate documentation for `second` with binary dependency on `first` to reproduce the issue.

Note: to test this, I was manually set version in [TestedVersions](https://github.com/Kotlin/dokka/blob/5f26eae85784b558243b879cbee98321562cd51c/dokka-integration-tests/gradle/src/main/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt#L17) to 2.1.0-Beta2.
I will prepare separate PR to test if we work fine with 2.1.0-Beta2